README.md

```diff
@@ -1,5 +1,9 @@
 ## OIDC Gatekeeper Operator - a component of the Charmed Kubeflow distribution from Canonical
 
+[![CharmHub Badge](https://charmhub.io/oidc-gatekeeper/badge.svg)](https://charmhub.io/oidc-gatekeeper)
+[![PR tests](https://github.com/canonical/oidc-gatekeeper-operator/actions/workflows/on_pull_request.yaml/badge.svg)](https://github.com/canonical/oidc-gatekeeper-operator/actions/workflows/on_pull_request.yaml)
```

Comment:
I'm thinking we can exclude the on pr tests, iiuc from our discussion that's the requirement.

Reply:
ACK, taking it out